data_model/src/isi.rs

```diff
@@ -84,13 +84,6 @@ pub struct RemoveKeyValueBox {
     pub key: EvaluatesTo<Name>,
 }
 
-/// Sized structure for all possible Sets.
-#[derive(Debug, Clone, Serialize, Deserialize, Encode, Decode, PartialEq, Eq, IntoSchema)]
-pub struct SetBox {
```

Comment:
never used?

Reply:
Two uses total. Here and in Schema. I would ask the library developers if they use it. If they don't I'd get rid of it.